docs/en/setup/examples/cases/agent-sensor/readme.md

```diff
@@ -0,0 +1,50 @@
+# Detect Process by Agent Sensor
+
+This documentation helps you to set up service which could let rover detected automatically.
+
+## Configure Service
+
+Configure your service to enable the Process Status Hook feature. Take [go2sky](https://github.com/SkyAPM/go2sky) as an example here.
```

Comment:
Please declare the minimum version here.

Reply:
This feature of go2sky haven't released. 1.5.0 is fine?

Reply:
I think so.

Reply:
updated.